thunderstorm/src/main/frontend/components/TS_Toggle/TS_Toggle.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {ComponentSync} from '../../core/ComponentSync';
+import {_className} from '../../utils/tools';
+import './TS_Toggle.scss';
+import {voidFunction} from '@nu-art/ts-common';
+
+type Props = {
+	id: string; // to make sure it's unique
```

Comment:
should be optional - you can generate one to bind the input to the label

Reply:
should I do this now or I can merge first?